integration/network/service_test.go

```diff
@@ -26,6 +26,7 @@ func delInterface(t *testing.T, ifName string) {
 }
 
 func TestDaemonRestartWithLiveRestore(t *testing.T) {
+	skip.If(t, testEnv.OSType == "windows")
```

Comment:
Actually; looks like none of these tests are run on Windows, so perhaps these skips should be removed, and a build-tag (and/or rename the file to `_linux_test.go`).

OTOH, a `skip` will show in the test logs (giving an indication how many tests were skipped 🤔

Reply:
Ok, I'll leave them in place.